src/bin/vip-wp.js

```diff
@@ -252,13 +252,40 @@ const bindReconnectEvents = ( {
 	} );
 };
 
+// Command examples
+const examples = [
+	{
+		usage: `vip @example-app.develop -- wp site list`,
+		description:
+			'Retrieve the list of network sites on the develop environment of the "example-app" WordPress multisite application.',
```

Comment:
```suggestion
			'Use a WP-CLI command to retrieve the list of network sites on the develop environment of the "example-app" WordPress multisite application.',
```

I'd prefix this with "Use a WP-CLI command" because it's using the WP-CLI via the VIP-CLI that we are documenting. We may want to use this prefix for the other descriptions in this block.

Reply:
Agreed!